src/coreclr/src/jit/simdashwintrinsiclistxarch.h

```diff
@@ -0,0 +1,126 @@
+//                  ISA          Function name               NumArg                                                                                                                                                                                                      Instructions                                                                                                                                                                                                                                           Flags
+//                                                                      {TYP_BYTE,                                  TYP_UBYTE,                                  TYP_SHORT,                                  TYP_USHORT,                                 TYP_INT,                                    TYP_UINT,                                   TYP_LONG,                                   TYP_ULONG,                                  TYP_FLOAT,                                  TYP_DOUBLE}
+// *************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
+//  Vector2 Intrinsics
+SIMD_AS_HWINTRINSIC(Vector2,     Abs,                        1,         {NI_Illegal,                                NI_Illegal,                                 NI_Illegal,                                 NI_Illegal,                                 NI_Illegal,                                 NI_Illegal,                                 NI_Illegal,                                 NI_Illegal,                                 NI_Vector2_Abs,                             NI_Illegal},                                SimdAsHWIntrinsicFlag::None)
+SIMD_AS_HWINTRINSIC(Vector2,     Equals,                     2,         {NI_Illegal,                                NI_Illegal,                                 NI_Illegal,                                 NI_Illegal,                                 NI_Illegal,                                 NI_Illegal,                                 NI_Illegal,                                 NI_Illegal,                                 NI_SSE_CompareEqual,                        NI_Illegal},                                SimdAsHWIntrinsicFlag::None)
```

Comment:
`Equals`, `GreaterThan`, `GreaterThanOrEqual`, `LessThan`, and `LessThanOrEqual` which return a vector are only for `Vector<T>` am removing them.